samples/Mvc.Client/Startup.cs

```diff
@@ -60,7 +60,14 @@ public void ConfigureServices(IServiceCollection services)
                 options.EnterpriseDomain = Configuration["GitHub:EnterpriseDomain"];
                 options.Scope.Add("user:email");
             })
-
+            /*
+            .AddKloudless(options =>
```

Comment:
We don't generally add new ones anymore (unless they're unusual or very popular) as with 60+ providers now if we included them this file would be huge 😄

Reply:
I wanted to add an example, because if you don't provide a scope it won't work and there is no "default scope" as it really depends on what you want to use in kloudless.
Is there a better way to provide an example ?

Reply:
You could add a Kloudless-specific document here: https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers/tree/dev/docs

Reply:
Added documentation and set default scope (like it's done in Deezer provider)